<?xml version="1.0"?>

<launch>
	<arg name="robot_name" default="robot1"/>

	<group ns="$(arg robot_name)">
		<node name="interfacer" pkg="robomaster_interface" type="pub_data.py" />
		<include file="$(find rplidar_ros)/launch/rplidar_a2m12.launch"/>
		<include file="$(find robot_pose_ekf)/robot_pose_ekf.launch" />
		<node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" args="0 0 0 0 0 0 base_link laser 30" />
		<node pkg="tf" type="static_transform_publisher" name="imu_broadcaster" args="0 0 0 0 0 0 base_link imu 30" />
		<node pkg="tf" type="static_transform_publisher" name="base_link_broadcaster" args="0 0 0 0 0 0 base_footprint base_link 30" />
	</group>

	
</launch>
